v3/plugins/eclipse/che-theia/7.3.1/meta.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: v2
 publisher: eclipse
 name: che-theia
-version: latest
+version: 7.3.1
 type: Che Editor
 displayName: theia-ide
-title: Eclipse Theia for Eclipse Che
-description: Eclipse Theia
+title: Eclipse Theia development version.
+description: Eclipse Theia, get the latest release each day.
```

Comment:
This title and description seems to be inaccurate, no? The version will be fixed here.

Reply:
good catch